### DISCUSSION TITLE
BT-702(a)-notice and listName attribute

### DISCUSSION THREAD

**Discussion by teslavitas** [OP-TED/eForms-SDK#609] [Q&A] [answered]
In SDK 1.8 field BT-702(a)-notice has a type "code".
Why is there no listName attribute in sample notices XML?
`<cbc:NoticeLanguageCode>FRA</cbc:NoticeLanguageCode>`

**Comment by YvesJo** [comment, marked as answer]
This is a design option. The listName is not expected for all fields of type code. 
Some fields of type code are associated to XML elements of another type (e.g. BT-727-Lot) which therefore do not have a listName attribute.
Regarding BT-702, there are two fields BT-702(a)-notice and BT-702(b)-notice which are associated respectively to an element of type code and an element of type identifier, the second one does not have a listName attribute. Specification of the listName attribute for the first field could have been possible; this does not however provide further relevant information. For consistency between the 2 fields it was decided to not expect the listName attribute for the first field.
With SDK 1.9 you'll be able to identify directly from fields.json the attributes to be specified.
